src/main/java/org/reaktivity/nukleus/http2/internal/stream/Http2ServerFactory.java

```diff
@@ -1131,8 +1187,10 @@ private void onNetworkData(
             DataFW data)
         {
             final long traceId = data.traceId();
-            final long authorization = data.authorization();
             final long budgetId = data.budgetId();
+            authorization = data.authorization();
+
+            counters.dataFramesRead.getAsLong();
```

Comment:
I think that counter is intended for `h2` `data` frames, not generic reaktivity `DATA` frames.

Reply:
I see